### PR TITLE
Fix aidl file path for different versions

### PR DESCRIPTION
**Pull request by Dexus**

### CODE REVIEW

**Comment by Dexus** [comment]
It fails on different versions of cordova so bring both path in the src to make it working

File: plugin.xml [hunk 1]
```diff
@@ -92,7 +92,8 @@ SOFTWARE.
         </config-file>
 
         <!-- In-app Billing Library -->
-		<source-file src="src/android/com/android/vending/billing/IInAppBillingService.aidl" target-dir="src/aidl/com/android/vending/billing" />
+	<source-file src="src/android/com/android/vending/billing/IInAppBillingService.aidl" target-dir="src/com/android/vending/billing" />
+	<source-file src="src/android/com/android/vending/billing/IInAppBillingService.aidl" target-dir="src/aidl/com/android/vending/billing" />
```
Comment (steveetm) [inline plugin.xml]: It should be ```<source-file src="src/android/com/android/vending/billing/IInAppBillingService.aidl" target-dir="app/src/main/aidl/com/android/vending/billing" />```.
Also, not really a nice solution but I couldn't came up better idea either. Also this should be merge ASAP, it is already late weeks ago...

**Comment by aesculus** [comment]
I am stilling getting an error in PhoneGap Build with this version.

Installing "cc.fovea.cordova.purchase" at "7.1.1" for android

and then I get the error still pointing to com

:app:compileReleaseJavaWithJavac/app/src/main/java/com/smartmobilesoftware/util/IabHelper.java:33: error: package com.android.vending.billing does not exist

**Comment by steveetm** [comment]
If you need fast fix, clone this plugin locally and do as I said above, use this for aidl:
`<source-file src="src/android/com/android/vending/billing/IInAppBillingService.aidl" target-dir="app/src/main/aidl/com/android/vending/billing" />`
instead of 
`<source-file src="src/android/com/android/vending/billing/IInAppBillingService.aidl" target-dir="src/com/android/vending/billing" />`

Honestly, I was dumb as *** as I pressed approve, it was clearly a request change:P

**Comment by j3k0** [comment]
@steveetm Do you think this fix will break things for people using older version of Cordova?

**Comment by steveetm** [comment]
It shouldn't break anything, only an unused directory. I'll retest it today, I worked on cordova-android@7 lately, but I have to move bug for some maintance on our cordova-android@6. I'll update on this soon.

**Comment by aesculus** [comment]
Confirmed works with PGB on cli-8.0.0.

**Pull request merged**